application-task-ui/src/main/resources/TaskManager/TaskManagerTranslations.xml

```diff
@@ -230,7 +230,14 @@ taskmanager.incompleteTasks.notification.fix.done=Data inferred succesfully
 taskmanager.incompleteTasks.notification.fix.error=Error while trying to infer data
 taskmanager.incompleteTasks.notification.fixAll.inprogress=Starting inferring process..
 taskmanager.incompleteTasks.notification.fixAll.done=Inferring process started
-taskmanager.incompleteTasks.notification.fixAll.error=Failed to start inferring process</content>
+taskmanager.incompleteTasks.notification.fixAll.error=Failed to start inferring process
+
+## Pickers translations
+TaskManager.TaskManagerProjectPickerDefault=None
+TaskManager.TaskManagerSpacePickerModalTitle=Space Picker
```

Comment:
Unusual translation key format. Why not stick with a standard format such as `taskmanager.spacepicker.modal.title=Space Picker`.

As for `TaskManager.TaskManagerProjectPickerDefault=None`, you can go for `TaskManager.TaskManagerClass_project_empty=None`

Reply:
I wasn't sure what was the right format.